Model already exists: ../sfd-main/src/cifar10/edm-cifar10-32x32-uncond-vp.pkl
Loading the pre-trained diffusion model from "../sfd-main/src/cifar10/edm-cifar10-32x32-uncond-vp.pkl"...
cherish
4
/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/torch/distributed/distributed_c10d.py:4631: UserWarning: No device id is provided via `init_process_group` or `barrier `. Using the current device set by the user. 
  warnings.warn(  # warn only once
Total parameters in U-Net:      60417164
Setting up optimizer...
Training for 200 kimg...

[rank0]: Traceback (most recent call last):
[rank0]:   File "/teamspace/studios/this_studio/sfd-main/train.py", line 155, in <module>
[rank0]:     main()
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/click/core.py", line 1161, in __call__
[rank0]:     return self.main(*args, **kwargs)
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/click/core.py", line 1082, in main
[rank0]:     rv = self.invoke(ctx)
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/click/core.py", line 1443, in invoke
[rank0]:     return ctx.invoke(self.callback, **ctx.params)
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/click/core.py", line 788, in invoke
[rank0]:     return __callback(*args, **kwargs)
[rank0]:   File "/teamspace/studios/this_studio/sfd-main/train.py", line 150, in main
[rank0]:     training_loop.training_loop(**c)
[rank0]:   File "/teamspace/studios/this_studio/sfd-main/training/training_loop.py", line 275, in training_loop
[rank0]:     teacher_traj = [loss_fn.get_teacher_traj(net=net_copy, tensor_in=latents[k], labels=labels[k]) for k in range(num_acc_rounds)]
[rank0]:   File "/teamspace/studios/this_studio/sfd-main/training/training_loop.py", line 275, in <listcomp>
[rank0]:     teacher_traj = [loss_fn.get_teacher_traj(net=net_copy, tensor_in=latents[k], labels=labels[k]) for k in range(num_acc_rounds)]
[rank0]:   File "/teamspace/studios/this_studio/sfd-main/training/loss.py", line 142, in get_teacher_traj
[rank0]:     teacher_traj = self.solver_tea(
[rank0]:   File "/teamspace/studios/this_studio/sfd-main/solvers.py", line 573, in dpm_pp_sampler
[rank0]:     denoised = get_denoised(net, x_cur, t_cur, class_labels=class_labels, condition=condition, 
[rank0]:   File "/teamspace/studios/this_studio/sfd-main/solvers.py", line 15, in get_denoised
[rank0]:     denoised = net(x, t, class_labels=class_labels, step_condition=step_condition)
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1751, in _wrapped_call_impl
[rank0]:     return self._call_impl(*args, **kwargs)
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1762, in _call_impl
[rank0]:     return forward_call(*args, **kwargs)
[rank0]:   File "/teamspace/studios/this_studio/sfd-main/models/networks_edm.py", line 602, in forward
[rank0]:     D_x = c_skip * x.repeat(1, self.model.repeat) + c_out * F_x
[rank0]: RuntimeError: Number of dimensions of repeat dims can not be smaller than number of dimensions of tensor
